### PR TITLE
invites are now working

### PR DESCRIPTION
**Pull request by tomassasovsky**

### CODE REVIEW

**Comment by tomassasovsky** [comment]
moves to code review #5

File: lib/src/email_sender/email_sender.dart [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  }) async {
+    final email = Message()
+      ..from = Address(
+        Constants.inviteEmailAccount,
+        Constants.inviteEmailUserName,
+      )
+      ..recipients.add(to)
+      ..subject = 'Invitation to join the ${organization.name} organization'
+      ..html =
+          "<h1>Here's your invite link:</h1>\n<p>https://${Constants.host}/invite/$refId</p>\n<p>This link will expire in one week.</p>\n<p>If you don't want to join the organization, you can ignore this email.</p>\n<h2>$message</h2>\n<p>Thanks,</p>\n<p>${Constants.inviteEmailUserName}</p>";
```
Comment (NazarenoCavazzon) [inline lib/src/email_sender/email_sender.dart]: Maybe be should create a function that returns the string, and we pass that function the variables to make it easy to change

File: lib/src/invite/invite_service.dart [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    );
+
+    if (invite == null || invite.isEmpty) {
+      return null;
+    }
+
+    return Invite.fromJson(invite);
+  }
+
+  Future<Invite?> findInviteByRefId({
```
Comment (NazarenoCavazzon) [inline lib/src/invite/invite_service.dart]: This Method is not used

**Review by NazarenoCavazzon — COMMENTED** [review]
Everything looks fine

**Review by NazarenoCavazzon — COMMENTED** [review]
All good

**Review by NazarenoCavazzon — APPROVED** [review]

**Pull request merged**